uaclient/status.py

```diff
@@ -233,6 +233,11 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_REFRESH_SUCCESS = "Successfully refreshed your subscription"
 MESSAGE_REFRESH_FAILURE = "Unable to refresh your subscription"
 
+ERROR_ON_ALLOW_BETA_KEY = """\
+Misspelled value for allow_beta key: {user_key} in \
```

Comment:
Probably better to say "Invalid value" instead of "Misspelled value".
Also it's better not to hardcode static errors as we can give more information.

Let's also say
Expected a boolean string, but found: {invalid_value}."""